src/constants/errors/index.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+export const INVALID_FORM_NAME:string = '0';
```

Comment:
why are you setting them as numbers?

Reply:
It could literally be anything. Makes no difference, as we use the constants on both ends.